packages/schematics/angular/utility/latest-versions/package.json

```diff
@@ -3,6 +3,7 @@
   "comment": "This file is needed so that dependencies are synced by Renovate.",
   "private": true,
   "dependencies": {
+    "@angular/core": "^18.0.0-next.0",
```

Comment:
Won't this cause renovate to continually change this version now?

Reply:
No, since it’s unpinned